mongo/options/loggeroptions.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) MongoDB, Inc. 2023-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package options
+
+import (
+	"go.mongodb.org/mongo-driver/internal/logger"
+)
+
+// LogLevel is an enumeration representing the supported log severity levels.
+type LogLevel int
+
+const (
+	// LogLevelInfo enables logging of informational messages. These logs
+	// are High-level information about normal driver behavior.
```

Comment:
Once the logging implementation is more finalized, it would be good to enumerate either here or elsewhere on what the `Info` or `Debug` level will actually log for each component.

Reply:
That's probably a good idea, the documentation might get fairly large over time. Do you have a format in mind?

Reply:
I do not and I think you're right that the documentation may become too large to fit over a couple types or methods/functions. I think for now I'd just focus on the implementation and eventually recruit the docs team to create a page on https://www.mongodb.com/docs/drivers/go with more detailed information about logging.